Solutions/Corvus.Json.CodeGeneration.201909/Corvus.Json.CodeGeneration.Draft201909/Generators/CodeGenerator.Array.Add.cs

```diff
@@ -17,7 +16,7 @@ namespace Corvus.Json.CodeGeneration.Generators.Draft201909 {
     public partial class CodeGeneratorArrayAdd : CodeGeneratorArrayAddBase {
         
         
-        #line 123 "../../Corvus.Json.CodeGeneration.Abstractions/SharedTemplates/CodeGenerator.Array.Add.tt"
+        #line 123 "C:\Users\matth\source\repos\Corvus.JsonSchema\Solutions\Corvus.Json.CodeGeneration.Abstractions\SharedTemplates\CodeGenerator.Array.Add.tt"
```

Comment:
I just noticed this now has absolute paths to your hard drive. Could this be a problem for anyone other than you attempting to single-step through this sort of stuff using sourcelink?

Reply:
(Fixed by reverting to an older version of the mono-T4 tool. We have an open PR which fixes this problem on the mono-t4 repo, but there is no response from that team.